lib/v2/oncc/money18.js

```diff
@@ -0,0 +1,84 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const sections = {
+    exp: '新聞總覽',
+    fov: '全日焦點',
+    industry: '板塊新聞',
+    int: '國際金融',
+    recagent: '大行報告',
+    ntlgroup: 'A股新聞',
+    pro: '地產新聞',
+    weainvest: '投資理財',
+    ipo: '新股IPO',
+    tech: '科技財情',
+};
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id ?? 'exp';
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 30;
+
+    const rootUrl = 'https://money18.on.cc';
+    const currentUrl = `${rootUrl}/finnews/news_breaking.html?section=${id}`;
+    const apiUrl = `${rootUrl}/cnt/utf8/content/20230210/articleList/list_${id}_all.js`;
+    const ipoApiUrl = `https://dyn.on.cc/api/asrh/v1/events/names/新股/articles?page=1&limit=${limit}`;
+
+    const response = await got({
+        method: 'get',
+        url: id === 'ipo' ? ipoApiUrl : apiUrl,
+    });
+
+    let items = [];
+
+    if (id === 'ipo') {
+        items = response.data.articles.slice(0, limit).map((item) => ({
+            title: item.title,
+            author: item.authorname,
+            link: `${rootUrl}/finnews/content/${id}/${item.articleId}.html`,
+            description: art(path.join(__dirname, 'templates/money18.art'), {
+                images: item.hasHdPhoto ? [item.hdEnlargeThumbnail] : undefined,
```

Comment:
Please add `https://hk.on.cc/hk/bkn` to each img url string

Reply:
👌